common/src/main/java/com/spotify/spydra/util/StorageUtils.java

```diff
@@ -0,0 +1,82 @@
+package com.spotify.spydra.util;
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import com.google.api.gax.paging.Page;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.BlobInfo;
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.BucketInfo;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import java.util.Iterator;
+
+
+public class StorageUtils {
+    private static final Storage storage = StorageOptions.getDefaultInstance().getService();
+
+    public Bucket checkBucket(String bucketName) {
+        if (bucketName == null) {
+            throw new IllegalArgumentException(
```

Comment:
I think I prefer the pattern of [`Objects.requireNonNull(bucketName, 'Provide bucket name.')`](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#requireNonNull-T-java.lang.String-). Though this is fine too.